src/app/shared/components/template/components/task-card/task-card.component.ts

```diff
@@ -13,7 +13,8 @@ import { TemplateFieldService } from "../../services/template-field.service";
 export class TmplTaskCardComponent extends TemplateBaseComponent implements OnInit {
   style: string;
   highlighted: boolean;
-  highlightedText = "Active";
+  highlightedText: string;
+  subtasksText: string;
```

Comment:
nit(non-blocking)
I often try to avoid 'sub' as a prefix as it tends not to be a very extensible keyword (e.g. sub-subtasks). I find 'child' seems to work a bit better as devs reasonably understand the children of one component/element can then become parents of others, and children of children usually work fine defined just as 'child'. But that's just personal preference, no huge issue either way

Reply:
actually in second thought does it even need reference to the subtasks? would it make more sense as progressText or some other keyword that better defines what it's actually targeting? Is this just the word being used to describe the group of items? I'm sure there could be something a bit more specific e.g. progressGroupName or similar? Probably worth checking with authoring